parsel/selector.py

```diff
@@ -139,9 +139,9 @@ class Selector(object):
     selectorlist_cls = SelectorList
 
     def __init__(self, text=None, type=None, namespaces=None, root=None,
-                 base_url=None, _expr=None):
+                 base_url=None, _expr=None, parser_cls=None):
```

Comment:
Well, the idea was to support other custom subclasses of etree.HTMLParser (in the case of html5parser.HTMLParser, one could build a subclass of that supporting the encoding argument).
Does this make sense or do you prefer to drop it?
I'm not attached to it, so I'm fine with either way.